command/app.go

```diff
@@ -197,6 +197,7 @@ func Commands() []*cli.Command {
 	return []*cli.Command{
 		NewListCommand(),
 		NewCopyCommand(),
+		NewPipeCommand(),
```

Comment:
Could you move this line under NewCatCommand()? It makes sense to show `cat` and `pipe` together.

Reply:
Resolved. https://github.com/peak/s5cmd/pull/587/commits/c563a044e1c8e075e79c5c71b9317fd9fac7beb9